### ISSUE TITLE
[BUG] - BBB-Exporter reporting offline in grafana

### ISSUE THREAD

**Issue by BrutalBirdie** [ebbba-org/ansible-role-bigbluebutton#172]
# TL;DR
The `bbb_exporter_secrets.env` file has the IP instead of the hostname.
When generating this file `bbb-conf --secret` is used to get the secret and URL.

When executing `bbb-conf --secret` without sudo this is the output:
```
$USER@3b ~ $ bbb-conf --secret # (EDITOR NOTE - I removed the username)
Permission denied: /etc/bigbluebutton/turn-stun-servers.xml
Permission denied: /etc/bigbluebutton/turn-stun-servers.xml
failed to load external entity "/etc/bigbluebutton/turn-stun-servers.xml"
grep: /etc/bigbluebutton/bbb-web.properties: Permission denied
grep: /etc/bigbluebutton/bbb-web.properties: Permission denied
grep: /etc/bigbluebutton/bbb-web.properties: Permission denied
grep: /etc/bigbluebutton/bbb-web.properties: Permission denied
grep: /etc/bigbluebutton/bbb-web.properties: Permission denied

URL: http://$PUBLICIP/bigbluebutton/ # (EDITOR NOTE - I removed the public ip)
Secret: REDACTED

Link to the API-Mate:
https://mconf.github.io/api-mate/#server=http://$PUBLICIP/bigbluebutton/&sharedSecret=REDACTED
```
This error only occurs when a different user is used then root to deploy this role.

# Tracing the ERROR

Part affected is monitoring.

https://github.com/ebbba-org/ansible-role-bigbluebutton/blob/2f40d1ffb8cf177f1e0f03ddb667edc4247c9d94/tasks/main.yml#L85-L88
https://github.com/ebbba-org/ansible-role-bigbluebutton/blob/2f40d1ffb8cf177f1e0f03ddb667edc4247c9d94/tasks/monitoring/all_in_one.yml#L8-L17

https://github.com/ebbba-org/ansible-role-bigbluebutton/blob/2f40d1ffb8cf177f1e0f03ddb667edc4247c9d94/templates/monitoring/bbb_exporter_secrets.env.j2#L1-L2

https://github.com/ebbba-org/ansible-role-bigbluebutton/blob/83471e86d6ddf982cd7929fe71cdbbac87b8b8ad/tasks/runtimevars.yml#L8-L24

Narrow down:
https://github.com/ebbba-org/ansible-role-bigbluebutton/blob/83471e86d6ddf982cd7929fe71cdbbac87b8b8ad/tasks/runtimevars.yml#L8-L12

This part must be execute with `become: true`
